tests/test_cases/test_select_testcase/y_tests.py

```diff
@@ -3,4 +3,4 @@
 
 @cocotb.test()
 async def y_test(dut):
```

Comment:
If there's a second test with the same prefix (y_test_dos), and TESTCASE=y_test, the filter will match both. I don't think this is the expected behavior.

Reply:
This is the new expected behavior. It more closely resembles pytest's `-k` option. If you only want to match "exactly" use `^name$`. It's regular expressions.